pom.xml

```diff
@@ -30,7 +30,7 @@
   <properties>
     <revision>1.10</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.121.2</jenkins.version>
```

Comment:
I don't have any problem with that, but why do you need to upgrade the core requirement? I thought making sure to depend on `2.60.3` was the minimum.

Reply:
Because many of the Pipeline plugins were bumped to this min req:
![image](https://user-images.githubusercontent.com/223853/56193539-2613b300-6031-11e9-8a56-a324261c00a7.png)